doc/src/guides/contributing/docstring.rst

```diff
@@ -673,16 +673,16 @@ of objects that have conventions here:
 1. Objects that are included in ``from sympy import *``, for example,
 ``sympy.acos``.
 
-For these, use ``:obj:`~.acos()```. The ``~`` makes it so that
-the text in the rendered HTML only shows ``acos``. Without it, it would use the
-fully qualified name ``sympy.functions.elementary.trigonometric.acos``. However,
-for names that are part of the global ``sympy`` namespace, we do not want to
-encourage accessing them from their specific submodule, as this is an
-implementation detail that could change. The ``.`` makes it so that the function
-name is found automatically. Sometimes, Sphinx will give a warning that there
-are multiple names found. If that happens, replace the ``.`` with the full name.
-For example, ``:obj:`~sympy.solvers.solvers.solve()```. For functions, methods,
-and classes, it is a convention to add () after the name to indicate such.
+For these, use ``:obj:`~.acos()```. The ``~`` makes it so that the text in the
+rendered HTML only shows ``acos``. Without it, it would use the fully qualified
+name ``sympy.functions.elementary.trigonometric.acos``. For names that are part
+of the global ``sympy`` namespace, we want to encourage accessing them from
+``sympy.*``, not the specific submodule. The ``.`` makes it so that the
+function name is found automatically. Sometimes, Sphinx will give a warning
+that there are multiple names found. If that happens, replace the ``.`` with
+the full name.  For example, ``:obj:`~sympy.solvers.solvers.solve()```. For
+functions, methods, and classes, it is a convention to add () after the name to
+indicate such.
```

Comment:
```suggestion
For these, use ``:obj:`~.acos()```. The ``~`` makes it so that the text in the
rendered HTML only shows ``acos`` instead of the fully qualified name
``sympy.functions.elementary.trigonometric.acos``. (This will encourage importing
names from the global ``sympy`` namespace instead of a specific submodule.)
The ``.`` makes it so that the function name is found automatically. (If Sphinx gives
a warning that there are multiple names found, replace the ``.`` with
the full name.  For example, ``:obj:`~sympy.solvers.solvers.solve()```.) Adding a trailing
pair of parentheses is a convention for indicating the name is a function, method, or
class.
```